### PR TITLE
Update MAINTAINERS.md

### PR DESCRIPTION
**Pull request by osg-grafana**
This change removes removes Karen Miller, and includes at least one alternative, interim technical writer.

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -2,4 +2,5 @@
 
 Some parts of the codebase have other maintainers:
 - `docs`:
-  - @KMiller-Grafana - [@KMiller-Grafana ](https://github.com/@kmiller-grafana) ([Grafana Labs](https://grafana.com/))
+  - @osg-grafana - [@osg-grafana](https://github.com/@osg-grafana) ([Grafana Labs](https://grafana.com/))
+  - @knylander-grafana - [@knylander-grafana](https://github.com/@knylander-grafana) ([Grafana Labs](https://grafana.com/))
```
Comment (chaudum) [inline MAINTAINERS.md:6]: The Github URLs do not have an `@` in front of the username:

```suggestion
  - @osg-grafana - [@osg-grafana](https://github.com/osg-grafana) ([Grafana Labs](https://grafana.com/))
  - @knylander-grafana - [@knylander-grafana](https://github.com/knylander-grafana) ([Grafana Labs](https://grafana.com/))
```

**Review by chaudum — COMMENTED** [review]
Otherwise ok

**Review by slim-bean — APPROVED** [review]
LGTM

**Review by dannykopping — APPROVED** [review]
Thanks!

**Pull request merged**